CUSTOMIZATION.md

```diff
@@ -127,3 +127,29 @@ packages.  If you don't want to support a particular package (e.g. if
 you have no Emacs experience and don't want to be responsible for
 students who show up with Emacs as their only editor), you can comment
 out that particular `or_dependency`.
+
+## Updating the repository
+
+If for some reason you need to get changes from this repository
+into the clone of it with your workshop page,
+please follow the steps bellow:
+
+1.  Add the template repository as upstream:
+
+        $ git remote add upstream git@github.com:swcarpentry/workshop-template.git
```

Comment:
Possibly add that if you get permission denied (or whatever it was again), try with https?